Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -786,9 +786,9 @@ void BTDiagnostics::MergeBuffersForPlotfile (int i_snapshot)
                                                   + Buffer_FabHeader.FabName(0);
             // Existing snapshot Fab Header Filename
             std::string snapshot_FabHeaderFilename = snapshot_Level0_path + "/Cell_H";
-            std::string snapshot_FabFilename = amrex::Concatenate(snapshot_Level0_path+"/Cell_D_",m_buffer_flush_counter[i_snapshot], 5);
+            std::string snapshot_FabFilename = amrex::Concatenate(snapshot_Level0_path+"/Cell_D_",m_buffer_flush_counter[i_snapshot], m_file_min_digits);
```

Comment:
regression found by @RevathiJambunathan: AMReX writes `Cell_D_` and `DATA_` files, hard-coded to 5 digits at the moment.
We need to find out how to control it.